scripts/install_third_party_libs.py

```diff
@@ -211,6 +214,73 @@ def ensure_system_python_libraries_are_installed(package, version):
     install_backend_python_libs.pip_install_to_system(package, version)
 
 
+def rewrite_android_proto_files():
+    """Edit all android proto files as protobuf not
```

Comment:
not --> does not

I think this could be quite a bit clearer, too. I don't know what you mean by "not support config imports".

Reply:
Keeping this much only `"""Edit all android proto files."""`. 
Other details are left as a part of comment 

```
    # Since there is no simple configuration for imports when using protobuf to
    # generate Python files we need to manually fix the imports.
    # See: https://github.com/protocolbuffers/protobuf/issues/1491
```